core/src/test/java/edu/stanford/futuredata/macrobase/pipeline/CubePipelineTest.java

```diff
@@ -0,0 +1,23 @@
+package edu.stanford.futuredata.macrobase.pipeline;
+
+import edu.stanford.futuredata.macrobase.analysis.summary.Explanation;
+import spark.utils.IOUtils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.FileInputStream;
+
+public class CubePipelineTest {
+    @Test
+    public void testInlineCsv() throws Exception{
+
+        FileInputStream inputStream = new FileInputStream("demo/sample_inlinecsv.json");
+        String jsonString = IOUtils.toString(inputStream);
```

Comment:
Prefer using one of the more standard libraries like Guava or Apache commons to convert this, I think we already have them as dependencies

Reply:
I couldn't get either apache commons or guava to work right. Apache.commons.io wasn't being found for import and decided not to mess with build dependencies.